package.json

```diff
@@ -43,6 +41,5 @@
     "commitizen": {
       "path": "cz-conventional-changelog"
     }
-  },
-  "version": "1.0.1"
```

Comment:
Yeah, semantic-release will ensure the correct one is published. This was causing some confusion.